Header header

float32 exposure_ms

float32 pid_value
float32 pid_error
float32 pid_p_term
float32 pid_i_term
float32 pid_d_term
float32 pid_pid_term
